requirements.txt

```diff
@@ -1,4 +1,3 @@
-ansible==2.8.3
```

Comment:
Shouldn't let ansible requirement for `ansible` part of this file for every use not running Docker ?
 
In the docker file, running a task like `pip install --upgrade ansible=xxx` to override ansible version installed by `requirements.txt`

If we removed `ansible` from the file, other requirements came with ansible and their version might change with based on ansible version installed on user's system:

- `cffi`
- `cryptography`
- `enum34`
- `ipaddress`
- `Jinja2`
- `MarkupSafe`
- `pycparser`
- `PyYAML`
- `six`

Reply:
@titom73 I agree.  I wasn't thinking of the non-docker scenarios.  Not adding ansible to the requirements.txt file will definitely break these instances.